src/RpcServer/RpcServer.cs

```diff
@@ -49,6 +48,7 @@ public RpcServer(NeoSystem system, RpcServerSettings settings)
             Initialize_SmartContract();
         }
 
+
```

Comment:
Like the variable name change in LogReader.cs, these kind of changes make it harder to review PRs. We should do changes like this in a dedicated PR like https://github.com/neo-project/neo/pull/2785